README.md

```diff
@@ -1,68 +1,39 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Donator
+By: @92yo, @francisA2000, @LinaQh92
 
-## Available Scripts
+## A place which makes your life a bit more easy when doing a good deed. Stop worrying about when and where and start looking for what to donate, and let us do the rest.
 
-In the project directory, you can run:
+---
 
-### `npm start`
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Topics
+* [Overview](#page_with_curl-overview)
+* [Installing locally](#floppy_diskpackage-installing-locally-)
+* [Our process](#construction_worker-construction---our-process)
+* [Our struggles](#hatching_chick-hatched_chick--our-struggles)
+* [Learning outcomes](#chicken-learning-outcomes)
+* [Limitations and future goals](#fast_forward--current-limitations-and-future-goals)
```

Comment:
so do you want to add this in this PR ? do you want to keep this pull request opened for work in progress

Reply:
its still work in progress, i wanna keep it open.